website/redirects.js

```diff
@@ -5212,4 +5172,468 @@ module.exports = [
     destination: "/consul/docs/monitor/telemetry/appdynamics",
     permanent: true,
   },
+  ///////////////////////////////////
+  // Additional 404 fixes.         //
+  ///////////////////////////////////
+  {
+    source: "/api-docs/:slug*",
+    destination: "/consul/api-docs/:slug*",
+    permanent: true,
+  },
+  {
+    source: "/commands/:slug*",
+    destination: "/consul/commands/:slug*",
+    permanent: true,
+  },
```

Comment:
Suggesting to remove these redirects as making them valid via prefixing their sources with `/consul` would make them redundant. More context in this comment: https://github.com/hashicorp/consul/pull/23003/files#r2482388755


```suggestion
```